form.js

```diff
@@ -245,27 +223,16 @@ define('form', ['doc'], function($) {
 		},
 
 		/**
-		 * @param label anchor
-		 * @param interpolated message
-		 *
-		 * Usage example:
-		 * 	form.appendMessage(<INPUT_LABEL_ANCHOR>, INTERPOLATED_MESSAGE);
-		 */
-		'appendMessage': function(label, message) {
-			var $label = $(label);
-			appendMessage($label, message);
-			addEventToClearErrorMessages($label.find('input'));
-		},
-
-		/**
-		 * @param form
+		 * @param $field
 		 *
 		 * Usage example:
-		 * 	form.removeValidationErrors(<FORM_ELEMENT>);
+		 * 	form.validateField(<FIELD_ELEMENT>);
 		 */
-		'removeValidationErrors': function(form) {
-			removeValidationErrors($(form));
+		'validateField': function($field) {
+			return {
+				message: getEmailValidate($field) || getUrlValidate($field) || getPatternValidate($field) || getMinValidate($field) || getMaxValidate($field) || getMaxLengthValidate($field) || getRequiredValidate($field) || '',
```

Comment:
How about do not send message when the field is valid? That way its easily to check if is valid or not.

@tisvasconcelos